ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectList/Index.cshtml.cs

```diff
@@ -41,11 +41,6 @@ public async Task OnGetAsync()
 		{
 			ApiResponse<ApiV2Wrapper<IEnumerable<AcademyConversionProject>>> response = await _repository.GetAllProjects(CurrentPage, _pageSize);
 
-			if (!response.Success)
```

Comment:
Removed this because (a) it does nothing and (b) "no results" is not exceptional.